### DISCUSSION TITLE
Update 3 - No Progress

### DISCUSSION THREAD

**Discussion by Mrbt0907** [CosmicHikari/Weather2-Remastered#14] [Announcements]
I've been absent for a good time now. I needed a break since I have been coding for 2 months in a row now.

I am going to take this week off then continue on the next week with the update as planned. I'll start posting images about the update if I make any progress from now on. Thank you guys for suggestions and bug reports.

**Comment by Justin7654** [comment]
Status update? it's been 22 days, Btw i really hopes this finishes this will be my favorite mod. also yes i just signed up for this so i can upvote it and like it

**Reply by Mrbt0907** [reply]
Making very slow progress atm. I got sick shortly after the last update. I'm working on finishing the new net code.
